pyvista/examples/downloads.py

```diff
@@ -1653,11 +1653,88 @@ def download_frog(load=True):  # pragma: no cover
     this dataset.
 
     """
-    # TODO: there are other files with this
     download_file('froggy/frog.zraw')
     return _download_and_read('froggy/frog.mhd', load=load)
 
 
+def download_frog_tissue(load=True):  # pragma: no cover
+    """Download frog tissue dataset.
+
+    This dataset contains tissue segmentation labels for the frog dataset
+    (see :ref:`download_frog`).
```

Comment:
```suggestion
    (see :func:`download_frog`).
```

Reply:
I'm not very familiar with how the references work for the docs... Is ``:func:`pyvista.examples.downloads.download_frog` `` preferred?